pkg/operator/baremetal_test.go

```diff
@@ -206,13 +206,13 @@ func TestGetMetal3DeploymentConfig(t *testing.T) {
 		t.Logf("Unstructed Config:  %+v", u)
 		t.Errorf("Failed to get Baremetal Provisioning Config from CR %s", testConfigResource)
 	}
-	actualCacheURL := getMetal3DeploymentConfig("CACHEURL", baremetalConfig)
+	actualCacheURL := getMetal3DeploymentConfig("CACHEURL", *baremetalConfig)
```

Comment:
I realize this is just a test, but there is a possible null pointer deref here.

Reply:
Yes. Fixing this in the follow-up PR.